app/Http/Controllers/DailyMenuController.php

```diff
@@ -110,4 +112,61 @@ public function store(DailyMenuCreateRequest $request)
         }
         return redirect()->route('daily-menus.create', ['date' => $date]);
     }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param string $date The date to get menu to show
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function show($date)
+    {
+        $menuOnDate = $this->dailyMenu->with('food.category')->where('date', $date)->paginate($this->dailyMenu->ITEMS_PER_PAGE);
+
+        return view('daily_menus.show', ['menuOnDate' => $menuOnDate, 'date' => $date]);
+    }
+
+    /**
+     * Update item in Menu List on Date
+     *
+     * @param UpdateMenuItemRequest $request The request message from Ajax request
+     *
+     * @return Response
+     */
+    public function update(UpdateMenuItemRequest $request)
+    {
+        $quantity = $request['quantity'];
+        $menuId = $request['menuId'];
+        $dailyMenu = $this->dailyMenu->find($menuId);
+        $error = __('Has error during update menu item');
+        $success = __('Update menu item success');
+
+        if ($dailyMenu->update(['quantity' => $quantity])) {
+            $result = [$dailyMenu->updated_at, 'message' => $success];
+            return response()->json($result, 200);
```

Comment:
Don't using hardcode, You should using 
`Response::HTTP_OK` replace for `200`
(**use Illuminate\Http\Response;**)